host-controller/src/main/java/org/jboss/as/host/controller/parsing/DomainXml_13.java

```diff
@@ -694,106 +602,6 @@ private void initializeRolloutPlans(ModelNode address, List<ModelNode> list) {
         list.add(addOp);
     }
 
```

Comment:
`writeAccessControl` can also be removed since this file will be used for read only.
https://github.com/wildfly/wildfly-core/pull/4228/files#diff-3a186e12b2e248de03d22e9991e1d13cR686